### DISCUSSION TITLE
What is underline within amino acid for CDR3?

### DISCUSSION THREAD

**Discussion by Theragen-Bio** [milaboratory/mixcr#1097] [Q&A]
I found the "_" underline within amino acid for CDR3.
For example, the nucleotide sequence for CDR3 was TGCAGTCCCTACGGGCAGATACTACGAGCAGTACTTC,
the putative amino acid from mixcr was CSPYGQ_YYEQYF, however, in silico translation was quite different "CSPYGQILRAVL".
Are there any special meaning for "_"?

**Comment by mizraelson** [comment]
Hi,
Could you please send the version of MiXCR, exact commands you used for the analysis, and MiXCR reports?
